test/metabase/pulse/render/body_test.clj

```diff
@@ -404,86 +404,6 @@
 (defn has-inline-image? [rendered]
   (some #{:img} (flatten-html-data rendered)))
 
-(defn- render-bar-graph [results]
```

Comment:
Is there a reason all of these tests were just removed? Seems like they should be ported over to the new system unless they aren't valid any more.